addons/ai/functions/fnc_garrison.sqf

```diff
@@ -4,7 +4,7 @@
  * Garrison function used to garrison AI inside buildings.
  *
  * Arguments:
- * 0: The building(s) nearest this position are used <POSITION>
+ * 0: The building(s) nearest this position are used <ARRAY>
```

Comment:
wouldnt `ARRAY of NUMBERs` be more consistent?

Reply:
Now that I think about it, I would interpret `<ARRAY of NUMBERs>` as a list consisting of any amount of numbers. `<ARRAY>` would be generic. Technically, we would need to write `<ARRAY<NUMBER, NUMBER, NUMBER>>` for a 3D position or an array containing exactly 3 numbers. I don't know what the best solution here would be.

Reply:
I think position is better than array, the biwiki uses position, and even specifies which kind (ASL, ATL)

Reply:
```suggestion
 * 0: The building(s) nearest this position are used <POSITION>
```